### ISSUE TITLE
missing FRIENDS class should report error

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#899]
https://github.com/larshp/abapGit/issues/3418

https://github.com/abapGit/build/tree/d7a9b283c4e46f82924858f1f5bb2d50bca1cd40

or perhaps something is missing in the abaplint.json setup in abapGit

**Comment by larshp** [comment]
minimal example, an error should be reported if the friend classes cannot be resolved

```abap
CLASS zcl_foobar DEFINITION CREATE PRIVATE FRIENDS zcl_injector.
ENDCLASS.
CLASS zcl_foobar IMPLEMENTATION.
ENDCLASS.
```

**Comment by larshp** [comment]
closing, moving to https://github.com/abaplint/abaplint/issues/1215
